CMakeLists.txt

```diff
@@ -109,6 +109,10 @@ add_compile_definitions(
     QT_USE_QSTRINGBUILDER
 )
 
+if (NOT USE_QT5)
```

Comment:
Similar to #700 wonder if this needs to use Qt5_FOUND or whether something like `QT_VERSION_MAJOR` could be compared?